plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/utils/TableViewerBuilder.java

```diff
@@ -50,13 +51,14 @@ public TableViewerBuilder(TableViewer viewer, Composite tableContainer) {
 		viewer.setComparer(new EqualityComparer());
 		this.tableLayout = new TableColumnLayout();
 		tableContainer.setLayout(tableLayout);
-		FontData fd = viewer.getTable().getFont().getFontData()[0];
+		normalFont = viewer.getTable().getFont();
+		FontData fd = normalFont.getFontData()[0];
 		fd.setStyle(SWT.ITALIC);
-		italic = new Font(null, fd);
+		italicFont = new Font(null, fd);
```

Comment:
I have no idea. But doesn't it make sense the default font for the table would actually be JFaceResources.DEFAULT_FONT?

I'm not fighting to impose using that approach anyway. Just letting you know it's a possibility. If your approach makes more sense, then keep at it.